publications/ohbm_2024/brainhack_project.md

```diff
@@ -0,0 +1,100 @@
+```
+
+# Other hubs covered by the leaders
+(Please choose other hubs that are covered by your leaders (be sure to have at least one leader in the specific timezones, and make a note next to their names))
+- Seoul
+- Hybrid (Asia / Pacific)
+- Hybrid (Europe / Middle East / Africa)
+- Hybrid (Americas)
+```txt
+# TODO: Bertrand for Hybrid (Europe)?, Remi for Hybrid (Americas)?
+```
+
+# Skills
+(Please provide skills you expect contributors to have, so others can have better ideas if they can join. You can include non-technical skills, or simply write None, check [the guide](https://github.com/ohbm/hackathon2024/blob/main/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-skills) for more suggestions.)
+```txt
+We welcome all contributions from various skill sets and levels. This can include opening discussions around improvements to the [documentation](https://nilearn.github.io/stable/index.html) and/or [code base](https://github.com/nilearn/nilearn
+), answering or commenting on questions or [issues raised on github](https://github.com/nilearn/nilearn/issues/4434) and [neurostars](https://neurostars.org/tag/nilearn), reviewing [pull requests](https://github.com/nilearn/nilearn/pulls), and [contributing code](https://nilearn.github.io/stable/development.html#how-to-contribute-to-nilearn).
+```
+
+# Recommended tutorials for new contributors
+(Please provide a list of tutorials that might be helpful for new contributors. Make sure to include the link to the tutorials here. Several great examples of such tutorials can be found at the following places: [traintrack online selection](https://ohbm.github.io/hackathon2023/traintrack/), [brainhack school modules](https://school-brainhack.github.io/modules/), [Neurohackademy lectures](https://neurohackademy.org/course_type/lectures/), and [brainhack's youtube channel](https://www.youtube.com/c/brainhackorg/playlists))
+```txt
+We recommend starting with [Nilearn's basic tutorials](https://nilearn.github.io/stable/auto_examples/00_tutorials/index.html#) and the [introduction to Nilearn](https://nilearn.github.io/stable/introduction.html). This would help new contributors get familiar with the package and its functionalities. They can even provide feedback on the tutorials and suggest improvements.
+```
+
+# Good first issues
+(If you welcome new contributors, please provide a list of good-first-issues (not required, but highly recommended), check [the guide](https://github.com/ohbm/hackathon2024/blob/main/.github/ISSUE_TEMPLATE/handbooks/projects.md#good-first-issues) for suggestions)
+```txt
+Depending upon your comfort level with the package, issues can be filtered as follows:
+- [Good first issues](https://github.com/nilearn/nilearn/labels/Good%20first%20issue)
+- [Effort: Low](https://github.com/nilearn/nilearn/labels/Effort%3A%20low)
+- [Effort: Medium](https://github.com/nilearn/nilearn/labels/Effort%3A%20medium)
+- [Effort: High](https://github.com/nilearn/nilearn/labels/Effort%3A%20high)
```

Comment:
I think it would be better to actually provide a list f these issues.

Reply:
Will shortlist some of the issues